### PR TITLE
[demo-app-mac] Fixed UI bugs  

### PR DESCRIPTION
**Pull request by AnatolyPristensky**
~~* [ ] Has `CHANGELOG.md` been updated?~~
* [x] Are tests passing locally?
* [x] Are the files formatted correctly?
~~* [ ] Did you add unit tests?~~
* [x] Did you check UI tests on the sample app? They are not executed on CI.
* [x] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description

1. UserID text field on AppCenter tab now doesn't expanding vertically.
2. Fixed behaviour of "Enter UserID" pop-up window. "Save" button now became default button instead of "Reset".
3. Fixed empty column in Properties table in Analytics tab.
4. Fixed duplicated buttons in tables view in Transmission tab.
5. Implemented new method to close app by pressing Close button (ObjC and Swift).

### CODE REVIEW

File: SasquatchMac/SasquatchMac/ViewControllers/AppCenterViewController.swift [hunk 1]
```diff
@@ -227,27 +228,26 @@ class AppCenterViewController : NSViewController, NSTextFieldDelegate, NSTextVie
 
   @IBAction func setUserID(_ sender: NSButton) {
     let alert: NSAlert = NSAlert()
+
```
Comment (AnastasiaKubova) [inline SasquatchMac/SasquatchMac/ViewControllers/AppCenterViewController.swift:231]: Please add a comment or remove empty lines (same for 236 and 421 lines)

File: SasquatchMac/SasquatchMac/ViewControllers/AppCenterViewController.swift [hunk 2]
```diff
@@ -55,6 +55,7 @@ class AppCenterViewController : NSViewController, NSTextFieldDelegate, NSTextVie
     setEnabledButton?.state = appCenter.isAppCenterEnabled() ? .on : .off
     networkRequestsAllowed?.state = appCenter.isNetworkRequestsAllowed() ? .on : .off
     setAppSecretButton?.isEnabled = startUpModeForCurrentSession == StartupMode.AppCenter.rawValue || startUpModeForCurrentSession == StartupMode.Both.rawValue
+
```
Comment (AnastasiaKubova) [inline SasquatchMac/SasquatchMac/ViewControllers/AppCenterViewController.swift:58]: Remove pls. empty line

File: SasquatchMac/SasquatchMacSwift/AppDelegate.swift [hunk 3]
```diff
@@ -19,7 +19,9 @@ enum StartupMode: Int {
 @NSApplicationMain
 @objc(AppDelegate)
 class AppDelegate: NSObject, NSApplicationDelegate, CrashesDelegate, CLLocationManagerDelegate {
-
```
Comment (AnastasiaKubova) [inline SasquatchMac/SasquatchMacSwift/AppDelegate.swift:22]: evert it pls and add an empty line after the new method

File: SasquatchMac/SasquatchMacObjC/AppDelegate.m [hunk 4]
```diff
@@ -19,7 +19,9 @@ @interface AppDelegate ()
 enum StartupMode { appCenter, oneCollector, both, none, skip };
 
 @implementation AppDelegate
-
```
Comment (AnastasiaKubova) [inline SasquatchMac/SasquatchMacObjC/AppDelegate.m:22]: evert it pls and add an empty line after the new method

File: SasquatchMac/SasquatchMac/ViewControllers/TransmissionViewController.swift [hunk 5]
```diff
@@ -298,38 +298,41 @@ class TransmissionViewController: NSViewController, NSTableViewDataSource, NSTab
 
     // Transmission target section
     if let cell = tableView.makeView(withIdentifier: NSUserInterfaceItemIdentifier(rawValue: "target"), owner: nil) as? NSTableCellView {
+        for subview in cell.subviews {
```
Comment (AnastasiaKubova) [inline SasquatchMac/SasquatchMac/ViewControllers/TransmissionViewController.swift:301]: formatting

File: SasquatchMac/SasquatchMac/ViewControllers/AppCenterViewController.swift [hunk 6]
```diff
@@ -227,27 +228,26 @@ class AppCenterViewController : NSViewController, NSTextFieldDelegate, NSTextVie
...
+    let textView: NSTextField = NSTextField(frame: NSRect(x: 0, y: 0, width: 200, height: 25))
+    textView.stringValue = UserDefaults.standard.string(forKey: kMSUserIdKey) ?? ""
+    alert.accessoryView = textView
     alert.alertStyle = .warning
+    
     switch(alert.runModal()) {
     case .alertFirstButtonReturn:
-      UserDefaults.standard.removeObject(forKey: kMSUserIdKey)
-      appCenter.setUserId(nil)
+        let text = textView.stringValue
```
Comment (AnastasiaKubova) [inline SasquatchMac/SasquatchMac/ViewControllers/AppCenterViewController.swift:244]: formatting

File: SasquatchMac/SasquatchMac/ViewControllers/AppCenterViewController.swift [hunk 7]
```diff
@@ -227,27 +228,26 @@ class AppCenterViewController : NSViewController, NSTextFieldDelegate, NSTextVie
...
+        UserDefaults.standard.set(text, forKey: kMSUserIdKey)
+        appCenter.setUserId(text)
       break
     case .alertSecondButtonReturn:
-      let text = textView.string
-      UserDefaults.standard.set(text, forKey: kMSUserIdKey)
-      appCenter.setUserId(text)
+        UserDefaults.standard.removeObject(forKey: kMSUserIdKey)
```
Comment (AnastasiaKubova) [inline SasquatchMac/SasquatchMac/ViewControllers/AppCenterViewController.swift:249]: formatting

File: SasquatchMac/SasquatchMac/ViewControllers/AppCenterViewController.swift [hunk 8]
```diff
@@ -228,27 +228,23 @@ class AppCenterViewController : NSViewController, NSTextFieldDelegate, NSTextVie
...
     alert.addButton(withTitle: "Cancel")
-    let scrollView: NSScrollView = NSScrollView(frame: NSRect(x: 0, y: 0, width: 300, height: 40))
-    let textView: NSTextView = NSTextView(frame: NSRect(x: 0, y: 0, width: 290, height: 40))
-    textView.string = UserDefaults.standard.string(forKey: kMSUserIdKey) ?? ""
-    scrollView.documentView = textView
-    scrollView.hasVerticalScroller = true
-    scrollView.contentView.scroll(NSPoint(x: 0, y: textView.frame.size.height))
-    alert.accessoryView = scrollView
-    alert.alertStyle = .warning
+    let textView: NSTextField = NSTextField(frame: NSRect(x: 0, y: 0, width: 200, height: 25))
```
Comment (DmitriyKirakosyan) [inline SasquatchMac/SasquatchMac/ViewControllers/AppCenterViewController.swift:234]: Rename the var to "textField"

File: SasquatchMac/SasquatchMacSwift/AppDelegate.swift [hunk 9]
```diff
@@ -20,6 +20,11 @@ enum StartupMode: Int {
 @objc(AppDelegate)
 class AppDelegate: NSObject, NSApplicationDelegate, CrashesDelegate, CLLocationManagerDelegate {
 
+  // Enable closing app by pressing Close button.
+  func applicationShouldTerminateAfterLastWindowClosed(_ sender: NSApplication) -> Bool {
+         return true
+    }
```
Comment (AnastasiaKubova) [inline SasquatchMac/SasquatchMacSwift/AppDelegate.swift:26]: formatting

**Review by DmitriyKirakosyan — APPROVED** [review]

**Pull request merged**